src/features/metadata/metadataApi.ts

```diff
@@ -20,10 +20,10 @@ export const getCachedMetadataLocalStorageKey = (chainId: number): string =>
 export const getAllTokens = async (chainId: number) => {
   let tokens;
   if (!tokensCache[chainId]) {
-    tokensCache[chainId] = fetchTokens(chainId);
+    tokensCache[chainId] = (await fetchTokens(chainId)).tokens;
+    //handle failure here, need to decide what to do with errors
```

Comment:
Please prefix with `TODO` and link to https://github.com/airswap/airswap-web/issues/201